### DISCUSSION TITLE
Pulsetimer that stays off for certain amount of time

### DISCUSSION THREAD

**Discussion by martin072** [arendst/Tasmota#16285] [Q&A] [answered]
I am wondering if a feature exists on Tasmota that, when turning an output OFF with Pulsetimer after x seconds, it would also be possible to leave it off for x seconds disregarding any input signals for that output?

I have a small waterpump that is allowed to run for 20min continuously, but then needs to be off for at least 2 min for cooldown. It's triggered by a float sensor, hence the above question. I know it can be done with rules or scripts, just wondering if there is anything already existing for it, or if not, if this would be a feature that can be created?

Thanks

**Comment by sfromis** [comment, marked as answer]
No, it is not the type of common scenario already existing without doing it with rules etc, which are meant to handle that type of situation.
